lib/services/pets/firebase_pets_service.dart

```diff
@@ -9,7 +9,9 @@ class FirebasePetsService extends PetsService {
 
   @override
   Future<Pet> addPet(Pet newPet) async {
-    await _firestore.collection("pets").add(newPet.toJson());
+    await _firestore
+        .collection("pets")
+        .add(newPet.toJson());
     return newPet;
   }
```

Comment:
The pet Id is never saved inside the Pet entity like we do for Users, how do we do updates and deletes then?

Reply:
For editing or deleting, the owner needs first to navigate to the pet-list, where pets are objects with ids, got from Firebase...